frontend/backend/ofrak_server.py

```diff
@@ -330,6 +338,37 @@ async def delete_comment(self, request: Request) -> Response:
         )
         return web.json_response(await self._serialize_component_result(result))
 
+    @exceptions_to_http(SerializedError)
+    async def search_for_vaddr(self, request: Request) -> Response:
+        resource = await self._get_resource_for_request(request)
+        vaddr_start, vaddr_end = self._serializer.from_pjson(
+            await request.json(), Tuple[int, Optional[int]]
+        )
+        try:
+            if vaddr_end is not None:
+                matching_resources = await resource.get_descendants(
+                    r_filter=ResourceFilter(
+                        attribute_filters=(
+                            ResourceAttributeRangeFilter(
+                                Addressable.VirtualAddress, vaddr_start, vaddr_end
+                            ),
+                        )
+                    ),
+                    r_sort=ResourceSort(Addressable.VirtualAddress),
+                )
+            else:
+                matching_resources = await resource.get_descendants(
+                    r_filter=ResourceFilter(
+                        attribute_filters=(
+                            ResourceAttributeValueFilter(Addressable.VirtualAddress, vaddr_start),
+                        )
+                    ),
+                    r_sort=ResourceSort(Addressable.VirtualAddress),
+                )
```

Comment:
In my opinion, this is a little bit easier to read:

```suggestion
            if vaddr_end is not None:
                filter = ResourceAttributeRangeFilter(Addressable.VirtualAddress, vaddr_start, vaddr_end)
            else:
                filter = ResourceAttributeValueFilter(Addressable.VirtualAddress, vaddr_start),
            matching_resources = await resource.get_descendants(
                r_filter=ResourceFilter(attribute_filters=(filter,)),
                r_sort=ResourceSort(Addressable.VirtualAddress),
            )
```

Reply:
I applied a similar change, with different name for `filter` so it doesn't overlap the builtin